test/extensions/network/dns_resolver/cares/dns_impl_test.cc

```diff
@@ -434,20 +545,50 @@ class DnsImplTest : public testing::TestWithParam<Address::IpVersion> {
   DnsImplTest()
       : api_(Api::createApiForTest()), dispatcher_(api_->allocateDispatcher("test_thread")) {}
 
+  envoy::config::core::v3::TypedExtensionConfig getTypedDnsResolverConfig(
+      const std::vector<Network::Address::InstanceConstSharedPtr>& resolver_inst,
+      const envoy::config::core::v3::DnsResolverOptions& dns_resolver_options) {
+    envoy::config::core::v3::TypedExtensionConfig typed_dns_resolver_config;
+    envoy::extensions::network::dns_resolver::cares::v3::CaresDnsResolverConfig cares;
+    auto dns_resolvers = envoy::config::core::v3::Address();
+
+    // setup and address
```

Comment:
Did you mean `setup the address`?
Should this be `// Setup the resolvers.`?

Reply:
ACK